### PR TITLE
Add Ansible Basics deck

### PR DESCRIPTION
**Pull request by beeankha**
Adding a new deck, plus including it in the http://ansible.github.io/lightbulb/ index page.

### CODE REVIEW

**Comment by thedoubl3j** [comment]
this presentation was also given at fest

File: index.html [hunk 1]
```diff
@@ -49,6 +49,9 @@ <h2>Training Decks</h2>
             <div class="col-sm-4">
                 <a class="btn btn-secondary btn-block" href="/lightbulb/decks/ansible-best-practices.html">Ansible Best Practices</a>
             </div>
+            <div class="col-sm-4">
+                <a class="btn btn-secondary btn-block" href="/lightbulb/decks/ansible-best-practices.html">Ansible Best Practices</a>
```
Comment (gundalow) [inline index.html]:
```suggestion
                <a class="btn btn-secondary btn-block" href="/lightbulb/decks/ansible_basics.html">Ansible Basics</a>
```

File: decks/ansible_basics.html [hunk 2]
```diff
@@ -0,0 +1,737 @@
...
+            <li>3500+ GitHub Contributors</li>
+            <li>Over 1500 modules shipped with Ansible</li>
+            <li>New contributors added every day</li>
+            <li>1400+ users on IRC channel</li>
+            <li>World-wide meetups taking place every week</li>
+            <li>Ansible Galaxy: over 18,000 subscribers</li>
+            <li>500,000+ downloads a month</li>
+            <li>AnsibleFests, Ansible Automates, and other global events</li>
+          </ul>
+          <p><font color="52AAAC"><strong>http://ansible.com/community</strong></font><br></p>
```
Comment (gundalow) [inline decks/ansible_basics.html]:
```suggestion
          <p><font color="52AAAC"><strong>http://ansible.com/community</strong></font><br></p>
```
```suggestion
          <p><font color="52AAAC"><strong>https://ansible.com/community</strong></font><br></p>
```

File: index.html (hunk 1 above)
Comment (beeankha) [reply index.html]: Committed the above change!

File: index.html [hunk 3]
```diff
@@ -49,6 +49,9 @@ <h2>Training Decks</h2>
             <div class="col-sm-4">
                 <a class="btn btn-secondary btn-block" href="/lightbulb/decks/ansible-best-practices.html">Ansible Best Practices</a>
             </div>
+            <div class="col-sm-4">
+                <a class="btn btn-secondary btn-block" href="/lightbulb/decks/ansible-basics.html">Ansible Best Practices</a>
```
Comment (nqb) [inline index.html]:
```suggestion
                <a class="btn btn-secondary btn-block" href="/lightbulb/decks/ansible-basics.html">Ansible Basics</a>
```

Comment (nqb) [reply index.html]: Change title.

File: decks/ansible_basics.html [hunk 4]
```diff
@@ -0,0 +1,737 @@
...
+            <li>1400+ users on IRC channel</li>
+            <li>World-wide meetups taking place every week</li>
+            <li>Ansible Galaxy: over 18,000 subscribers</li>
+            <li>500,000+ downloads a month</li>
+            <li>AnsibleFests, Ansible Automates, and other global events</li>
+          </ul>
+          <p><font color="52AAAC"><strong>https://ansible.com/community</strong></font><br></p>
+          <aside class="notes">
+            <p>Plus Ansible has an amazing community.  We have tons of contributors and active participants on IRC and in our Google project list. We have about 500,000 downloads a month, which is really exciting to know that so many people use this automation tool.
+              I’d actually like to take this opportunity to mention this year’s AnsibleFest going on during the first week of October in Austin, TX, as well as numerous Ansible Automates events happening all over the world and in the US during the last three months of this year.  If you can attend any of those, we’d love to see you there!
```
[the remark this answers is not in the document]
Comment (beeankha) [reply decks/ansible_basics.html]: Removed.

File: index.html (hunk 3 above)
Comment (beeankha) [reply index.html]: Thanks for catching that, @nqb!

**Pull request merged**